src/module.api/address.controller.ts

```
@@ -30,6 +31,51 @@ export class AddressController {
   ) {
   }
 
+  /**
+   * @param {string} address to list participate account history
+   * @param {PaginationQuery} query
+   */
+  @Get('/history')
+  async listAccountHistory (
+    @Param('address') address: string,
+      @Query() query: PaginationQuery): Promise<ApiPagedResponse<AddressHistory>> {
+    const limit = query.size > 30 ? 30 : query.size
```

Have reject large `query.size` to prevent attack.

---

hmm... do you mean throw error for large size instead of silently default??

---

My bad, I thought it was `query.size === undefined ? : 30 : query.size`
I am okay with capped `size`